opsin-core/src/main/resources/uk/ac/cam/ch/wwmm/opsin/resources/carbohydrates.xml

```diff
@@ -29,9 +29,12 @@ NOTE:A suffixAppliesTo is expected on all carbohydrates that may be effected by
 		<token value="O=CCC[C@H](O)[C@H](O)C" labels="/1/2/3/4//5//6" suffixAppliesTo="2">amicet</token>
 		<token value="O=C[C@H](O)C[C@@H](O)[C@@H](O)C" labels="/1/2//3/4//5//6" suffixAppliesTo="2" naturalEntIsOpposite="yes">ascaryl</token>
 		<token value="O=CC[C@H](O)[C@@H](O)[C@H](O)C" labels="/1/2/3//4//5//6" suffixAppliesTo="2">boivin</token>
+		<token value="O=CC[C@H](OC)[C@@H](O)[C@H](O)C" labels="/1/2/3///4//5//6" suffixAppliesTo="2">sarment</token>
 		<token value="O=C[C@@H](O)C[C@H](O)[C@@H](O)C" labels="/1/2//3/4//5//6" suffixAppliesTo="2" naturalEntIsOpposite="yes">colit</token>
 		<token value="O=CC[C@H](O)[C@H](O)[C@H](O)C" labels="/1/2/3//4//5//6" suffixAppliesTo="2">digitox</token>
+		<token value="O=CC[C@H](OC)[C@H](O)[C@H](O)C" labels="/1/2/3///4//5//6" suffixAppliesTo="2">cymar</token>
 		<token value="O=C[C@@H](O)[C@H](O)[C@H](O)[C@@H](O)C" labels="/1/2//3//4//5//6" suffixAppliesTo="2" naturalEntIsOpposite="yes">fuc</token>
+		<token value="O=C[C@@H](O)[C@H](OC)[C@H](O)[C@@H](O)C" labels="/1/2//3///4//5//6" suffixAppliesTo="2" naturalEntIsOpposite="yes">digital</token>
```

Comment:
The sources I can find indicate that when unspecified this is the D- enantiomer e.g. https://www.ebi.ac.uk/chebi/searchId.do?chebiId=10368 so the  naturalEntIsOpposite="yes" attribute isn't appropriate

Reply:
I might have messed up when I copied the patch sorry. Roger just sent me a random diff which I couldn’t apply directly and likely I missed any subtleties like that. On holiday atm so will sort when back

Reply:
I expect it's going to be a mistake on Roger's copy as well. digitalose is a methylated version of D-fucose, so it would be reasonable to use fucose as a starting point (it's defined in the line above)... with the only subtlety being that fucose typically means L-fucose.